src/materialized/src/bin/materialized/main.rs

```diff
@@ -187,7 +187,7 @@ fn run() -> Result<(), anyhow::Error> {
     if cfg!(debug_assertions) && !popts.opt_present("dev") && !ore::env::is_var_truthy("MZ_DEV") {
         bail!(
             "refusing to run dev (unoptimized) binary without explicit opt-in\n\
-             hint: Pass the '--dev' option or set the MZ_DEV environment variable to opt in.\n\
+             hint: Pass the '--dev' option or set the environment variable MZ_DEV=1 to opt in.\n\
```

Comment:
Seems reasonable, but I contend this is no longer grammatical. What about "...or set MZ_DEV=1 in your environment to opt in."

Reply:
That does read a bit better, but has the slight defect that someone who's a newbie at Unix conventions will have a harder time googling it than "how to set an environment variable".

Though maybe it doesn't matter, because making it easy to use Materialize for such people is a lost cause anyway? (And they probably would have installed from apt or downloaded the .tgz anyway)

Reply:
Yeah if that sentence doesn't make sense to you you're gonna have a hard time with other parts of Materialize.